package.json

```diff
@@ -45,7 +45,7 @@
     "helmet": "4.6.0",
     "is-url": "^1.2.4",
     "joi": "^17.4.0",
-    "mailgun-js": "^0.22.0",
+    "mailgun.js": "^3.5.4",
```

Comment:
Why do we change from `mailgun-js` to `mailgun.js`?

Reply:
can we bump the version instead of changing the library?

Reply:
there is a vulnerability issue in mailgun-js, because it is using netmask library as dependencies. there is a fix for this, but it is not updated in the npm lib.

therefor we try to used another lib, that is not using netmask.